.circleci/config.yml

```diff
@@ -128,6 +128,33 @@ jobs:
             - ~/.ccache
             - ~/.cache/pip
 
+  arm64:
+    machine:
+      image: ubuntu-2004:202101-01
+    resource_class: arm.medium
+    environment:
+      - CYTHON_VERSION: 'latest'
+      - JOBLIB_VERSION: 'latest'
+      - THREADPOOLCTL_VERSION: 'latest'
+      - PYTEST_VERSION: 'latest'
+      - PYTEST_XDIST_VERSION: 'latest'
+      - TEST_DOCSTRINGS: 'true'
```

Comment:
We should also set `OMP_NUM_THREADS=2` to avoid oversubcriptions problems if the docker container runs on a machine with many CPUs on the host but only 2 of them are allowed for the build as is common on most CI hosts.

Reply:
Related issue: #16016 but this could also happen with numpy/scipy functions that use OpenBLAS threads and/or OpenMP.